django_project/base/models/location_type.py

```diff
@@ -0,0 +1,29 @@
+# coding=utf-8
+"""Location type definition.
+
+"""
+
+from django.db import models
+
+
+class LocationType(models.Model):
+    """Location type model."""
+
+    name = models.CharField(
+        max_length=100,
+        blank=False,
+    )
+
+    description = models.TextField(
+        blank=True,
+        null=True,
+    )
```

Comment:
What I'm missing here is something to capture which types of geometries would be accepted.

Reply:
Added here https://github.com/kartoza/healthyrivers/pull/95/commits/401f4c66ccb17748fcbed656ddb71fa263264df0